.tekton/kf-notebook-controller-pull-request.yaml

```diff
@@ -214,6 +214,9 @@ spec:
         value: $(params.dockerfile)
       - name: CONTEXT
         value: $(params.path-context)
+      # KFLUXSPRT-2401: rename the secret so it's not found, otherwise it breaks ubi8:minimal builds
```

Comment:
@jiridanek I noticed this, is it fine for rhds?

Reply:
Top of file in annotations says

```
... has(body.repository) && body.repository.full_name == "opendatahub-io/kubeflow"
```

So the tekton pipeline only runs on odh/Kubeflow, we should be good.